.ci/docker/build.sh

```diff
@@ -218,6 +220,16 @@ case "$image" in
     CONDA_CMAKE=yes
     TRITON=yes
     ;;
+  pytorch-linux-jammy-xpu-2024.0-py3)
```

Comment:
You will also need to add a line in 
```
https://github.com/pytorch/pytorch/blob/main/.github/workflows/docker-builds.yml#L38
```
In order to make sure docker build is working

Reply:
Thanks, added

Reply:
Ah, now this explained why I did see the usual missing ECR entry error when pushing a brand new docker image name.  I guess Andrey has added it https://us-east-1.console.aws.amazon.com/ecr/repositories/private/308535385114/pytorch/pytorch-linux-jammy-xpu-2024.0-py3?region=us-east-1, so retry should work now.

Reply:
Yes, he helped to do it.